scripts/tasks/src/api-extractor.ts

```diff
@@ -1,29 +1,13 @@
-import * as glob from 'glob';
 import * as path from 'path';
-import { apiExtractorVerifyTask, task, series, logger, TaskFunction } from 'just-scripts';
-import { ExtractorMessageCategory } from '@microsoft/api-extractor';
-import chalk from 'chalk';
-import { getTsPathAliasesConfig, getTsPathAliasesApiExtractorConfig } from './utils';
-import { getJustArgv } from './argv';
 
-const apiExtractorConfigs: Array<
-  [
-    configPath: string,
-    /**
-     * config name is created from <configName> suffix `api-extractor.<configName>.json`.
-     * @example
-     * `api-extractor.fast.json -> configName === fast`
-     *
-     * default behavior:
-     * `api-extractor.json -> configName === default`
-     */
-    configName: string,
-  ]
-> = glob
-  .sync(path.join(process.cwd(), 'config/api-extractor*.json'))
-  .map(configPath => [configPath, configPath.replace(/.*\bapi-extractor(?:\.(.*))?\.json$/, '$1') || 'default']);
+import type { ExtractorMessageCategory, ExtractorResult } from '@microsoft/api-extractor';
+import chalk from 'chalk';
+import * as glob from 'glob';
+import { ApiExtractorOptions, TaskFunction, apiExtractorVerifyTask, logger, series, task } from 'just-scripts';
+import type * as ApiExtractorTypes from 'just-scripts/src/tasks/apiExtractorTypes';
```

Comment:
these types are not exposed, nor do they match with api-extractor types. thus need of deep import 🚨